### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone :Portfolio: setup and mobile version skeleton
understood how to parse a Figma design to create a UI.
Flexbox to place elements in the page.
Build a personal portfolio site.
Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

**Comment by ibrahim777764** [comment]
please review my code,let me know if there any changes,thankyou

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </a>
+        <nav>
+            <img src="icon.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="section1">
+        <h1> Hey there. I’m Ibrahim </h1>
+        <h1>
+            <span>I’m a software developer</span>
+        </h1>
```
Comment (ShoiraTa) [inline index.html:27]: - [ ] OPTIONAL:  Headings are used to better understand the context of different parts of a page by Google. Therefore it is recommended to have one `<h1>` tag in a page. I suggest you to combine two   `h1` tags into one. If you want to give some style you can use `span` tags and define styles by classes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            <li>
+                <a href="https://medium.com/">
+                    <img src="medium.svg" alt="medium icon">
+                </a>
+            </li>
+        </ul>
+
+        <button type="button">
+            <img src="Disabled.png" alt="disabled button">
+        </button>
```
Comment (ShoiraTa) [inline index.html:66]: - [ ] OPTIONAL: This button has some border, I recommend you to remove it. 
Screenshot 2022-02-15 at 19 56 28

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @ibrahim777764 ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Semantic HTML ✔️  



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**